repos/system_upgrade/common/models/multipath.py

```diff
@@ -0,0 +1,66 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class MultipathInfo(Model):
+    """ Available information about multpath devices of the source system. """
+    topic = SystemInfoTopic
+
+    is_configured = fields.Boolean(default=False)
+    """
+    True if multipath is configured on the system.
+
+    Detected based on checking whether /etc/multipath.conf exists.
+    """
+
+    config_dir = fields.Nullable(fields.String())
+    """ Value of config_dir in the defaults section. None if not set. """
+
+
+class UpdatedMultipathConfig(Model):
+    """ Information about multipath config that needed to be modified for the target system. """
+    topic = SystemInfoTopic
+
+    path = fields.String()
```

Comment:
@MichalHe  nitpick (not blocking), from my experience it's better to have one msg with a list inside then mutliple msgs with a simple string (it's more compact for reading). if we expect this msg could be produced by multiple actors in future, then having one msg per one file does not make such a big difference. otherwise it's usually better to have one msg with a list.

Note that emitting new msg per each single item is implemented in several actors already, so keeping the decision on you what you prefer. Just adding my experience from debugging that using lists is otherwise a better pattern from my pov.